swap_meet/item.py

```diff
@@ -1,2 +1,33 @@
+import uuid  # Importamos el módulo para generar IDs únicos
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0, age=0):
+
+        if id is None:
+            self.id = uuid.uuid4().int
+        else:
+            self.id = id
+
+        self.condition = condition
+        self.age = age
+
+    def get_category(self):
+
+        return self.__class__.__name__
+
+    def __str__(self):
+
+        return f"An object of type {self.__class__.__name__} with id {self.id}."
```

Comment:
You wrote the implementation for `get_category` that returns `self.__class__.__name__`. We should invoke that method here so we can avoid repeating code. 

```suggestion
        return f"An object of type {self.get_category()} with id {self.id}."
```